### DISCUSSION TITLE
Brackets.next 2.1 release features discussion

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#100] [Feature Request]
This thread is to identify features to be shipped as the next release of brackets. Please comment and upvote features in this thread.

[See this project for details on the November release.](https://github.com/brackets-cont/brackets/projects/6)

**Comment by abose** [comment]
Should we import and modify Every-Scrub Extension for brackets core

**Comment by PunyTRexArms** [comment]
A feature to choose highlight colors in Live Preview?

**Comment by Hovhannisyan-K** [comment]
Split screen text compare

**Comment by xuronno** [comment]
Tailwind CSS intellisense, and terminal

**Comment by abose** [comment]
New features scheduled for 2.1:
1. Navigation bar with forwarding and back navigation.
2. Locate file in file tree button in navbar
![new navigation bar](https://user-images.githubusercontent.com/5336369/148080494-f9410ba7-b437-44a4-a6f0-4fd39975e7f3.gif)

**Comment by loid48** [comment]
Not sure es6 and/or es7 extensions are available or in the works.  However these would bring the Brackets platform up to the current level.

**Comment by Kpax7** [comment]
I think the most anticipated feature that its absence caused a lot of disappointment is the ability to search and replace escaped characters like \r and \n and \t , etc...
Please [resurrect this old issue here](https://github.com/adobe/brackets/issues/8093).

**Comment by gintares** [comment]
To make cash of changes and files to recover mistakes. 

For example,  file-A.php was open.
I , by mistake saved file file-A.php under wrong name file-B.txt
and file-B.txt under name  file-A.php, 
than i saved file-B.txt under right name file-B.txt, which overwrote  file-A.php contents.

I believe it is good to have possibility to recover these changes.

**Comment by mwhq** [comment]
Apple keeps telling me Brackets will not run on a future OS upgrade. Can someone ensure that Brackets stays compatible with Apple OS updates?

**Reply by PunyTRexArms** [reply]
Brackets is now called Phoenix Code. It's great. Try it. https://phcode.io/
